demo/chaincode/golang/auction.go

```diff
@@ -11,6 +11,7 @@ import (
 	"fmt"
 
 	"github.com/hyperledger/fabric/core/chaincode/shim"
+	cid "github.com/hyperledger/fabric/core/chaincode/shim/ext/cid"
```

Comment:
I think the "cid" alias here is not needed here as it is the same as the package name. That is, below you call `cid.New(...` anyway.